homeTask_1/homeTask_1/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+
+namespace homeTask_1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //Console.WriteLine("Hello World!");
```

Comment:
Senseless comment

Reply:
Yes, I've fixed it. @EgorAntonovich would you mind reviewing this task?

Reply:
Yep. In my way it looks better.

Reply:
@EgorAntonovich Can I merge this feature into the main branch?